src/components/navbar/DropDown.tsx

```diff
@@ -9,13 +9,15 @@ import { serverSignOut } from "@/app/(auth)/authService";
 
 export default function DropDown({
   openState,
+  placeHolderVoyage,
 }: {
   openState?: boolean;
+  placeHolderVoyage?: string
 }) { 
   const dispatch = useAppDispatch();
   const allVoyages = useAppSelector((state) => state.user.voyageTeamMembers);
   const activeVoyage = allVoyages?.find((item) => item.voyageTeam.voyage.status.name === "Active");
-  const currentVoyage = activeVoyage?.voyageTeam.name ?? "Please join a voyage to see your status information.";
+  const currentVoyage = placeHolderVoyage ?? activeVoyage?.voyageTeam.name ?? "Please join a voyage to see your status information.";
```

Comment:
I don't think we should go with this approach. Have you seen this? https://storybook.js.org/tutorials/intro-to-storybook/react/en/data/

Reply:
Yeah I set it up like that before but kept getting 'not wrapped in provider error'. Probably rushed it and overlooked something obvious. I'll give it another go.